client/my-sites/domains/domain-management/components/email-verification/index.jsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+
+class EmailVerificationCard extends React.Component {
+	static propTypes = {
+		changeEmailHref: PropTypes.string,
+		contactEmail: PropTypes.string.isRequired,
+		verificationExplanation: PropTypes.array.isRequired,
+		resendVerification: PropTypes.func.isRequired,
+		selectedDomainName: PropTypes.string.isRequired,
+		selectedSiteSlug: PropTypes.string.isRequired,
+	};
+
+	state = {
+		submitting: false,
+		emailSent: false,
+	};
+
+	componentWillUnmount() {
+		if ( this.timer ) {
+			clearTimeout( this.timer );
+			this.timer = null;
+		}
+	}
+
+	revertToWaitingState = () => {
+		this.timer = null;
+		this.setState( { emailSent: false } );
+	};
+
+	handleSubmit = event => {
+		const { resendVerification, selectedDomainName } = this.props;
+
+		event.preventDefault();
+
+		this.setState( { submitting: true } );
+
+		resendVerification( selectedDomainName, error => {
+			if ( error ) {
+				this.props.errorNotice( error.message );
```

Comment:
This is `undefined` - it needs to `connect`ed.

Reply:
Er...yep.